langserver/internal/refs/refs.go

```diff
@@ -255,6 +262,16 @@ func (c *Config) defInfo(pkg *types.Package, info *types.Info, rootFile *ast.Fil
 	return nil, errors.New("no selector type")
 }
 
+// packageName returns name.Name if name != nil. Otherwise, it returns the last
+// element in importPath (e.g. "http" for "net/http")
+func packageName(importPath string, name *ast.Ident) string {
+	if name != nil {
+		return name.Name
+	}
+	split := strings.Split(importPath, "/")
+	return split[len(split)-1]
```

Comment:
That def. would be optimizing too soon, and probably for something that doesn't matter, BUT that reads a lot cleaner IMO so done :)